packages/echo-core/src/services/hookRegistry/hookRegistry.ts

```diff
@@ -2,9 +2,12 @@ import { BaseError } from '@equinor/echo-base';
 export interface EchoHookRegistry {
     registerHook: (hookRegistryItem: HookRegistryItem) => void;
     registerMultipleHooks: (hookRegistryList: HookRegistryItem[]) => void;
-    getHookByName: (hookName: RegisteredHookName) => EchoCustomHook;
+    getHookByName: (hookName: RegisteredHookName | string) => EchoCustomHook;
 }
 
+/**
+ * @deprecated This enum method should not be used. Use RegisteredHookName from EchoFramework instead.
+ */
```

Comment:
I think we should also remove useSetActiveCommPackNo and MC. We don't want anyone to start using these now that it's deprecated